css-canvas-medical/index.html

```diff
@@ -12,16 +12,16 @@
 <body>
   <div class="container">
     <div class="row">
-      <div class="column-half first-row-contains-logo-and-references">
-        <div class="logo-blue">
+      <div class="row logo-and-references">
+        <div class="row logo-blue">
```

Comment:
You shouldn't have a `row` immediately inside of a `row`. If the inner row is meant to be the full width of the outer row, you should have a setup like this:
```
row
  column-full
    row
```